pkg/process/process.go

```diff
@@ -14,30 +16,18 @@
    limitations under the License.
 */
 
-package proc
+package process
 
 import (
 	"context"
 	"io"
-	"sync"
 	"time"
 
 	"github.com/containerd/console"
+	"github.com/containerd/containerd/pkg/stdio"
+	"github.com/pkg/errors"
 )
 
-// Stdio of a process
-type Stdio struct {
-	Stdin    string
-	Stdout   string
-	Stderr   string
-	Terminal bool
-}
-
-// IsNull returns true if the stdio is not defined
-func (s Stdio) IsNull() bool {
-	return s.Stdin == "" && s.Stdout == "" && s.Stderr == ""
-}
-
 // Process on a system
 type Process interface {
```

Comment:
The `Process` interface is only used for Linux? Just want to confirm it

If so, the change is LGTM :)

Reply:
Good point. @crosbymichael - Why is this Linux only? I can see why the `runc` stuff should be moved to a process_unix.go file but the interface seems to be generic enough to allow Windows?

Reply:
updated